### ISSUE TITLE
Hi, would you know if it is possible, for a user, to have a checkbox so his profile is not listed in the People’s directory? We are concerned about GDPR and give the possibility to users to not be listed. It should be very simple to implement as we already have the “Profile visibility” 

### ISSUE THREAD

**Issue by tokao** [humhub/humhub#6097]

**Comment by tokao** [comment]
GDPR compliance for sites where humhub is part of a log in in a website and user should have the possibility to not being listed in the list of members.

**Comment by luke-** [comment]
Please use the template for new feature requests.
A short, meaningful issue title would also be good.

**Comment by dantefromhell** [comment]
Is this request related to #5535 ?

**Comment by luke-** [comment]
@dantefromhell No, Invisible Users are more like System Users.

**Comment by dantefromhell** [comment]
@luke- Sure I see how the use case is different. But I'm wondering if we actual implementation needs to be any different than an user being able to set the `visibility` flag for their own profile?

**Comment by luke-** [comment]
Yes, the visibility option that can be set for users themselves, should be more limited.
